src/CommunityVoices/App/Api/Controller/Quote.php

```diff
@@ -28,6 +28,29 @@ class Quote extends Component\Controller
         'contentCategories' => []
     ];
 
+    const META_DATA_FIELDS = [
+        'sourcetype',
+        'intervieweeorsourcedocument',
+        'organization',
+        'sponsororganization',
+        'topicthemeofinterview',
+        'urlsourcedocument',
+        'intervieweeemail',
+        'intervieweetelephone',
+        'urlinterviewconsent',
+        'urlt1survey',
+        'urlt2survey',
+        'urlinterviewtransription',
+        'urlinterviewarticle',
+        'dateapprovedbyinterviewee',
+        'urlphotographinterviewee',
+        'suggestedphotosource',
+        'suggestedphotoincv',
+        'createaslide'
+    ];
```

Comment:
Could you store this array within the `OberlinMetadata` model?  Otherwise, it seems hard to know when we will want to select other metadata fields.  I'm assuming we can start with a hardcoded metadata style set to Oberlin.

Reply:
@Sammidysam Sure, that makes sense. Do you just mean rename the variable?

Reply:
No, I am imagining we would have an entity named `OberlinMetadata` that would hold this information.  It is attached to a Quote, but I see it being still decently distinct.